back-end/controllers/users_controller.js

```diff
@@ -62,15 +65,15 @@ const login = (req, res) => {
                     expiresIn: process.env.TOKEN_EXPIRATION,
                 };
                 const token = jwt.sign(payload, process.env.SECRET, options);
-                console.log(token);
+                // console.log(token);
                 res.json(token);
             } else {
-                // res.status(422);
-                res.json({ error: "Invalid login check your password" });
+                res.status(422);
+                res.json([]);
             }
         } else {
-            // res.status(404);
-            res.json({ error: "Invalid login check your email" });
+            res.status(404);
+            res.json([]);
```

Comment:
Why are you returning an array?

Reply:
if email or password are incorrect return empty array  not jwt and in front end if response.data is empty array  Invalid User the user rewrite his email and password and cant redirect to home page